server/models/user.coffee

```diff
@@ -21,6 +21,9 @@ module.exports = User = cozydb.getModel 'User',
     owner: Boolean
     allow_stats: Boolean
     activated: Boolean
+    otp_key: String
```

Comment:
I don't understand: I'm already using snake case for the field names I've added.